.gitignore

```diff
@@ -10,3 +10,5 @@ tmp
 .classpath
 .project
 .settings
+*.swp
+tags
```

Comment:
We tend to just use `.*` in the purescript `.gitignore`s, not sure how you feel about that though... the only dotfiles we usually want are `.travis.yml` and `.gitignore` itself.